tests/test_facebook_archived_data.py

```diff
@@ -10,6 +10,10 @@ class FacebookArchivedData(FacebookBaseTest):
     def name():
         return "tap_tester_facebook_archived_data"
 
+    def streams_to_test(self):
```

Comment:
Super minor, but if you wanted to access these values as a property on the class instead of a method, you can use the `@property` decorator to this method (would result in `self.streams_to_test` instead of `self.streams_to_test()`).

Reply:
Is there any benefit to having this as a property instead of a method? Is it more pythonic/best practice or is there a functional reason?

Reply:
Not really a functional benefit in this case. It's helpful in other contexts since it also lets you define a setter and the like, which we don't need here.

The option stood out to me since it reads as a property (vs something like get_streams_to_test), so just a thought incase you wanted it to behave that particular way. But definitely a take it or leave it thing, just mentioned it in case it was helpful.

Reply:
Going to not implement the `@property` since everything is just called as a method right now. But I like that pattern. I think we should use it in the future. I want to play around with with abstract classes to sort of “force” specific implementations for SaaS taps, and will implement it there.